packages/cheqd-sdk/src/index.ts

```diff
@@ -0,0 +1,17 @@
+// Dids
+export {
+  CheqdDidRegistrar,
```

Comment:
Some are called `CheqdXXX` while others are called `CheqdSdkXXX`. I'm fine with `CheqdXX`, but we should make them consistent I think